examples/counter/Forc.lock

```diff
@@ -0,0 +1,14 @@
+[[package]]
+name = 'core'
+source = 'path+from-root-D8BC155605882939'
+dependencies = []
+
+[[package]]
+name = 'counter'
+source = 'root'
+dependencies = ['std']
+
+[[package]]
+name = 'std'
+source = 'path+from-root-D8BC155605882939'
+dependencies = ['core']
```

Comment:
We should not have individual lock files as examples folder is a workspace now. Did you manually add this file? How did it produced?

Reply:
I figured that by adding the `reverse_errors` field to the `BuildProfile` the idiomatic way for the user would be able to add it to their build profile instead of passing a flag, that way they can just run `forc build`. 
The lock file is being generated probably because I need to update.

Reply:
@kayagokalp

Reply:
Hey @gr00vytvniks, I am waiting you to remove the lock file to do a final review as it shouldn't be there. After removing it and building the example again, if it is generated again we might have a problem and we should deal with it before merging this one.

Reply:
Hey @kayagokalp I removed the lock file and it looks like I just had an outdated forc version. 👍